### PR TITLE
Feat(workflow): add apply application workflow step definition

### PR DESCRIPTION
**Pull request by FogDong**
**What this PR does / why we need it**:
add apply application workflow step definition

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
/cc @yangsoon

### CODE REVIEW

File: vela-templates/definitions/internal/apply-application.cue [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+"apply-application": {
+	type: "workflow-step"
+	annotations: {}
+	labels: {}
+	description: "Apply application for your workflow steps"
+}
+template: {
+	// apply application
+	ouput: op.#ApplyApplication & {}
```
Comment (hongchaodeng) [inline vela-templates/definitions/internal/apply-application.cue:13]: output?

**Review by yangsoon — APPROVED** [review]

File: pkg/stdlib/op.cue [hunk 2]
```diff
@@ -15,6 +15,23 @@ import (
...
+#ApplyApplication: #Steps & {
+	load: ws.#Load @step(1)
+	components: #Steps & {
+		for name, c in load.value {
+			"\(name)": #Steps & {
+				workload: kube.#Apply & {value: c.workload}
+				if c.auxiliaries != _|_ {
+					_key:  "trait.oam.dev/resource"
+					for index, o in c.auxiliaries {
+						"traits-\(o.metadata.labels[_key])": kube.#Apply & {value: o}
```
Comment (leejanee) [inline pkg/stdlib/op.cue:27]: suggest to use `traits: "\(o.metadata.labels[_key])"`, users can use components["component-name"].traits["outputs-name"] access the resource.

**Review by Somefive — APPROVED** [review]

**Review by leejanee — APPROVED** [review]
lgtm

**Review by hongchaodeng — APPROVED** [review]

**Pull request merged**